tests/test_core_schedule.py

```diff
@@ -1511,15 +1508,16 @@ def test_transforming_schedule_to_gtfs(schedule):
                       '2': float('nan'), '4': float('nan'), '1': float('nan')},
          'platform_code': {'5': float('nan'), '6': float('nan'), '7': float('nan'), '8': float('nan'),
                            '3': float('nan'), '2': float('nan'), '4': float('nan'), '1': float('nan')}}
-    )
-    assert_semantically_equal(
-        gtfs['routes'].to_dict(),
-        {'route_id': {0: 'service'}, 'route_short_name': {0: 'name_2'}, 'route_long_name': {0: ''},
-         'agency_id': {0: float('nan')}, 'route_desc': {0: float('nan')}, 'route_url': {0: float('nan')},
+    actual_stops = gtfs['stops'].to_dict()
+    assert_semantically_equal(expected_stops, actual_stops)
+    expected_routes = {'route_id': {0: 'service'}, 'route_short_name': {0: 'name_2'}, 'route_long_name': {0: ''},
```

Comment:
Strangely, I only had to replace NaN with None in expected routes, and not the other GTFS-related dictionaries. Not sure why this is the case, and did not investigate. Can you think of a reason, and does this seem like a problem?

Reply:
huh, weird! I would have thought that None would happen when you have a mixture of empty and non empty values in a column, but that's not the case here. I don't think it should make a difference though, pandas considers None and NaN empty, so if you do `fillna` both None and NaN values will be considered